container/id/id.go

```diff
@@ -17,10 +17,26 @@ const Size = sha256.Size
 //
 // ID is mutually compatible with github.com/nspcc-dev/neofs-api-go/v2/refs.ContainerID
 // message. See ReadFromV2 / WriteToV2 methods.
-//
-// Instances can be created using built-in var declaration.
 type ID [Size]byte
 
+// NewFromContainerBinary returns new ID calculated from the given NeoFS
+// container encoded into Protocol Buffers V3 with ascending order of fields by
+// number. It's callers responsibility to ensure the format of b. See
+// [container.Container.Marshal].
+func NewFromContainerBinary(b []byte) ID { return sha256.Sum256(b) }
```

Comment:
are you sure about naming? i read it more like ID(b) is happening inside

Reply:
> are you sure about naming?

quite understandable. Any suggestions from u?

Reply:
NewFromMarshalledContainer?

Reply:
oh, i read it one more time and got it wrong again. it just sounds to me like the `b` is a raw representation and here we are trying to decode it back. i would not expect hashing from just reading naming. so yes, `NewFromMarshalledContainer` sounds better to me

Reply:
btw, is it really used? do we need it? cant we get it from `Container`'s method?

Reply:
renamed